src/st/class-wpml-pb-loader.php

```diff
@@ -44,15 +46,7 @@ public function __construct(
 			}
 		}
 
-		if ( class_exists( 'WPML_Config_Built_With_Page_Builders' ) ) {
```

Comment:
This `WPML_Config_Built_With_Page_Builders` class is part of Core. I believe we had this check when the page builder packages were located in ST. Now it's not needed anymore.